ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/LfValueTranslation.scala

```diff
@@ -217,4 +248,31 @@ object LfValueTranslation {
 
   }
 
+  object ContractCache {
+
+    private implicit object `Key Weight` extends caching.Weight[Key] {
```

Comment:
Would it make sense to create a type parametrized weighting scheme for this instead? I.e., do you expect such weighting to be used for more caches than the 2 defined here?